worker.js

```diff
@@ -0,0 +1,19 @@
+const diff = require('./modules/diff')
```

Comment:
it looked to me at first that the split of `.worker` and `.route` was for testing locally vs the actual lambda deploy, but I'm confused about this file

Reply:
this is the collection of dedicated `worker` entrypoints specified through `serverless.yml`

Reply:
ooooh the reference is intact (in serverless it's `worker.xxx`)